### PR TITLE
Fix(DB): Remove hardcoded string

### PR DESCRIPTION
**Pull request by Yuziux**
## Changes Proposed:
-  Fixed hardcoded gossip to the DB
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4185

## SOURCE:

https://www.youtube.com/watch?v=2zflV8Lc5vg
https://www.wowhead.com/tbc/quest=10994/chasing-the-moonstone

## Tests Performed:

- Tested in-game, the gossip did yes show, Win-10.
- 

## How to Test the Changes:

1.  .tele azshara
2.  .q a 10994
3.  "Hand over the Southfury moonstone and I'll let you go." gossip 

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/scripts/Kalimdor/zone_azshara.cpp [hunk 1]
```diff
@@ -258,6 +256,7 @@ class npc_rizzle_sprysprocket : public CreatureScript
         void sGossipSelect(Player* player, uint32 /*sender*/, uint32 /*action*/) override
         {
             CloseGossipMenuFor(player);
+            me->ReplaceAllNpcFlags(NPCFlags(0));
```
Comment (Kitzunu) [inline src/server/scripts/Kalimdor/zone_azshara.cpp:259]: why

**Comment by Nyeriah** [comment]
@Yuziux It looks like some merge went wrong? If you need help with this PR feel free to throw us a message at Discord

**Comment by Yuziux** [comment]
- nah i dont want to contribute to this core anymore, thank you :D

> @Yuziux It looks like some merge went wrong? If you need help with this PR feel free to throw us a message at Discord

**Comment by Nyeriah** [comment]
Okay, I am sorry that you feel that way. But thanks for the interest in contributing. If you ever change your mind, you can hit me up on discord and we can talk about those issues. See ya and safe travels 🙏🏻

**Pull request closed without merging**